src/modules/site-v2/templates/_scripts/utils.js

```diff
@@ -24,7 +24,15 @@ function toggle_input(idOrEl, val) {
 
 
 function fetch_json(url) {
-  return fetch(url).then(res => res.json())
+  return new Promise((resolve, reject) => {
+    fetch(url).then(res => {
+      if (res.status == 404) {
```

Comment:
the inverted logic might be safer -- only resolve when we have status===200. (Otherwise, server can send 5XX and other 4XX) codes.